test/service/AltitudeService.test.js

```diff
@@ -0,0 +1,51 @@
+import { AltitudeService } from '../../src/services/AltitudeService';
+import { loadBvvAltitude } from '../../src/services/provider/altitude.provider';
+
+describe('AltitudeService', () => {
+
+	const setup = (provider = loadBvvAltitude) => {
+		return new AltitudeService(provider);
+	}; 
+
+	describe('init', () => {
+
+		it('initializes the service', async () => {
+			const instanceUnderTest = setup();
+			expect(instanceUnderTest._altitudeProvider).toBeDefined();
+			expect(instanceUnderTest._altitudeProvider).toEqual(loadBvvAltitude);
+		});
+
+		it('initializes the service with default provider', async () => {
+			const instanceUnderTest = new AltitudeService();
+			expect(instanceUnderTest._altitudeProvider).toEqual(loadBvvAltitude);
+		});
```

Comment:
Both tests use the default provider.
Shouldn't be the first test against a custom provider?